docusaurus/docs/Android/03-ui/04-components/02-channel-list-header.md

```diff
@@ -1,17 +1,51 @@
 # Channel List Header
 
-<!-- TODO: Import whatever makes sense to import from https://getstream.io/chat/docs/android/channel_list_header_view/?language=kotlin -->
 
-## Overview
+`ChannelListHeaderView` is a component that shows the title of the channels list, current connection status, avatar of the current user, 
+and provides an action button which can be used to create a new conversation. 
+It is designed to be displayed at the top of the channels screen of your app. 
+It consists of the following elements: _User avatar_, _Title view_, _Action button_.
+ 
+ | Light Mode | Dark Mode |
+ | --- | --- |
+ |![Light_mode](../../assets/channels_header.png)|![Dark_mode](../../assets/channels_header_dark.png)|
+ 
+## Usage
 
-<!-- TODO: Brief description and a couple screenshots with default styling. -->
-
-## Handling Actions
+To use `ChannelListHeaderView` in your layout, include it in your XML layout as shown below:
+```XML
+<io.getstream.chat.android.ui.channel.list.header.ChannelListHeaderView
+    android:id="@+id/channelListHeaderView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    />
+```
 
-<!-- TODO: This section -->
+`ChannelListHeaderView` is supposed to work with `ChannelListHeaderViewModel`. 
+The basic setup of the ViewModel and connecting it with the view can be done in the following way:
+ ```kotlin
+// Instantiate the view model 
+val channelListHeaderViewModel: ChannelListHeaderViewModel by viewModels()
+// Bind the view model with ChannelListView 
+channelListHeaderViewModel.bindView(chanelHeaderView, viewLifecycleOwner)
+// Optional: setup listeners for user avatar and action button 
+chanelHeaderView.setOnActionButtonClickListener {
+    navigateSafely(R.id.action_homeFragment_to_addChannelFragment)
```

Comment:
Hm, `navigateSafely` is something only relevant to our sample app. IMO, better just to leave a comment that you can navigate to create channel screen here.

Reply:
Fixed